modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowNotifications.testcase

```diff
@@ -112,12 +112,46 @@ definition {
 	}
 
 	@description = "Verify that the user can open new user details by notifications"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOpenNewUserDetailsByNotifications {
+		property test.name.skip.portal.instance = "WorkflowNotifications#CanOpenNewUserDetailsByNotifications";
 
-		// TODO LRQA-70881 CanOpenNewUserDetailsByNotifications pending implementation
+		task ("Give configure Workflow to Single Approver for User") {
+			Workflow.openWorkflowListView();
 
+			Navigator.gotoNavItem(navItem = "Configuration");
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "User");
+		}
+
+		task ("When create a New User") {
+			WorkflowUser.disableCaptchaToCreateNewUserAndSendPassword();
+
+			User.logoutPG();
+
+			WorkflowAsset.createNewUserAccount(
+				newPassword = "password",
+				userEmailAddress = "user@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+		}
+
+		task ("Then view a created user's notification and details with the admin account") {
```

Comment:
"Then a notification is sent to the Admin User containing the new user's details"